### ISSUE TITLE
Heroku Postgres troubles …

### ISSUE THREAD

**Issue by fjenett** [motionbank/piecemaker2-api#119]
@dasantonym … did you get this one also?

> {"status":500,"message":"PG::ConnectionBad: PQconsumeInput() SSL error: decryption failed or bad record mac\n","param":null}

**Comment by dasantonym** [comment]
yes, and i guess it is related to a stale connection due to the introduction of concurrency when i switched to the puma server. will have a look!
